common/src/main/java/com/wynntils/core/features/overlays/Overlay.java

```diff
@@ -176,7 +176,7 @@ public float getRenderX(OverlayPosition position) {
         return switch (position.getHorizontalAlignment()) {
             case Left -> section.x1() + position.getHorizontalOffset();
             case Center -> (section.x1() + section.x2() - this.getWidth()) / 2 + position.getHorizontalOffset();
-            case Right -> (section.x2() + position.getHorizontalOffset() - this.getWidth());
+            case Right -> section.x2() - position.getHorizontalOffset() - this.getWidth();
```

Comment:
What is the benefit of this?

Reply:
I started this PR off by this change. After the work I done in OverlayPosition, I think we do not need this breaking change. Reverted this.